x/subscription/client/cli/tx_buy.go

```diff
@@ -11,31 +9,35 @@ import (
 	"github.com/spf13/cobra"
 )
 
-var _ = strconv.Itoa(0)
-
-func CmdSubscribe() *cobra.Command {
+func CmdBuy() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "subscribe [consumer] [index] [is-yearly]",
-		Short: "Subscribe to a service plan",
-		Args:  cobra.ExactArgs(3),
+		Use:   "buy [index] [optional: consumer] [optional: duration]",
```

Comment:
lets add an example, also since duration is uint64 and not time.Duration you have to specify units (months)